zebedee-reader/src/main/java/com/github/onsdigital/zebedee/search/indexing/content/ZebedeeContentIndexer.java

```diff
@@ -0,0 +1,203 @@
+package com.github.onsdigital.zebedee.search.indexing.content;
```

Comment:
You are opening several streams in this class that are not being closed. You need to wrap them in a `try-with-resources` block to ensure they are closed or bad things will happen.